### PR TITLE
docs: add instructions to migrate custom commands

### PR DESCRIPTION
**Pull request by kdmccormick**
Authored by @NeOneSoft , originally in: https://github.com/overhangio/cookiecutter-tutor-plugin/pull/8

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -123,6 +123,14 @@ The plugin API was upgraded from v0 to v1 in Tutor v13.2.0. This cookiecutter ge
         )
         hooks.Filters.CONFIG_OVERRIDES.add_items(list(config.get("overrides", {}).items()))
 
+  - In case the plugin has custom commands to be available from CLI, you will need to implement the CLI_COMMANDS filter according
+    to `Tutor's reference documentation <https://docs.tutor.overhang.io/reference/api/hooks/consts.html#tutor.hooks.Filters.CLI_COMMANDS>`__.
```
[the remark this answers is not in the document]
Comment (kdmccormick) [reply README.rst:127]: @NeOneSoft Thanks! How does it looks now?

[the remark this answers is not in the document]
Comment (NeOneSoft) [reply README.rst:127]: Thankyou @kdmccormick  for me looks marvelous!!!

**Comment by kdmccormick** [comment]
Excellent! Thanks a bunch for your contribution @NeOneSoft 😄

**Comment by NeOneSoft** [comment]
Thankyou @kdmccormick!!!  for your all great support!!!

**Pull request merged**